api/public/tests/features/transcribe_audio.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Transcribe audio data
+  Test the integration with audio transcription service providers
+
+  @stt
+  Scenario: Transcribe audio using Google
+    When Utterance "what time is it" is transcribed using Google's STT API
+    Then the request will be successful
+    And Google's transcription will be correct
+    And the device's last contact time is updated
+
+  @stt
+  Scenario: Transcribe audio using Assembly AI
+    When Utterance "what time is it" is transcribed using Mycroft's transcription service
+    Then the request will be successful
+    And the transcription will be returned to the device
+    And the device's last contact time is updated
```

Comment:
Would it be worth having an audio file with very clear and known speech in it so the behave test could cover the entire audio to response cycle?

Reply:
I am not sure I understand.  There is a .flac file used in the test.

Reply:
oh sorry ignore me, I somehow missed the step definition and assumed the `when` step was returning a mock "what time is it" utterance.